doc.html

```diff
@@ -44,30 +44,47 @@ <h2>3 Pre-requisites</h2>
 both service consumers or service provider implementors:
 <pre>
     - GeoJSON specifications (<a href="https://geojson.org/">https://geojson.org/</a>)
+</pre>
```

Comment:
remove the middle /pre pre to make a single pre block with the two bullet points